src/components/felles/dato.tsx

```diff
@@ -1,15 +0,0 @@
-import React from 'react';
-import { Normaltekst } from 'nav-frontend-typografi';
-import EMDASH from '../../utils/emdash';
-import { StringOrNothing } from '../../utils/felles-typer';
-import { formaterDato, isNullOrUndefined } from '../../utils';
-
-function Dato(props: { dato: StringOrNothing }) {
-	if (isNullOrUndefined(props.dato)) {
```

Comment:
Hvorfor ble dato.tsx fjernet? Er det ikke greit å ha en komponent som viser en fornuftig verdi  for datoen hvis den mangler?

Reply:
Jeg trodde den dato komponenten er veldig sivilstandsdato specific. I denne komponenten dato har vist med "Fra:" teksten som passer ikke til alle datoene og denne komponenten var brukt bare et sted og det var i Sivilstand selv. formaterDato() sjekker i funksjonen selv om dato er null or undefined og formaterer det videre da tenkte jeg å bruke det direkte i komponetene istedenfor å bruke det via <Dato> komponenten :)

Reply:
Det ble kun brukt 1 sted før, men nå så er det en del steder hvor man kunne brukt det hvis man legger på en prop for å endre teksten "fra:" til noe annet. Men det er ikke så mye kode i den komponenten uansett, så det er nok greit som du sier å bruke det direkte i komponenten. Hvis det dukker opp ekstra behov for å style datoene på en spesiell måte, f.eks endre på marginer osv så kan det være greit å legge til dato-komponenten igjen